src/components/Carousel.js

```diff
@@ -285,10 +285,10 @@ export default class Carousel extends Component {
      const arrowRightWidth = this.arrowRightNode && this.arrowRightNode.offsetWidth;
      const width = this.node.offsetWidth - (arrowLeftWidth || 0) - (arrowRightWidth || 0);
 
-     this.setState({
+     this.setState(() => ({
```

Comment:
Why do you need a function here, if you don't use previousState?

Reply:
for consistency